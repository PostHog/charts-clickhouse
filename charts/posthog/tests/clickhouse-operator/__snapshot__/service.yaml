the manifest should match the snapshot when using default values:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app: clickhouse-operator
      name: clickhouse-operator-metrics
      namespace: NAMESPACE
    spec:
      ports:
      - name: clickhouse-operator-metrics
        port: 8888
      selector:
        app: clickhouse-operator
