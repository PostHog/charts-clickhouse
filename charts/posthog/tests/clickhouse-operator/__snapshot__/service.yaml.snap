the manifest should match the snapshot when using default values:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app: clickhouse-operator
        clickhouse.altinity.com/chop: 0.18.4
      name: clickhouse-operator-metrics
      namespace: NAMESPACE
    spec:
      ports:
      - name: clickhouse-operator-metrics
        port: 8888
      selector:
        app: clickhouse-operator
